.gitignore

```diff
@@ -57,3 +57,6 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
+
+#yarn
+yarn.lock
```

Comment:
we should not ignore this. Is important to commit the lock file to ensure all team members have the exact same version of the dependencies

Reply:
I know we use npm but is important to spot if someone is using yarn to agreed in a package manager. Mix both causes issues and inconsistency between the versions of the dependencies